semantic-conventions/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ Please update the changelog as part of any significant pull request.
 
 ## Unreleased
 
+## v0.15.1
+
+- Move footnotes back together with the rendered table in Markdown
```

Comment:
This is a bit confusing since from the test it looks like they were completely gone. But I see that you also changed the test YAML. So the footnotes would have moved further away from the table? I think it's no big deal either way, but since unnecessary changes add noise in the spec repo, this reversion to the old behavior is good.

All in all, I still would love to see somebody tackle https://github.com/open-telemetry/opentelemetry-specification/issues/1925 to entirely get rid of this splintered output.

Reply:
Yes, it just moved it around and there was no test covering that before but they were not lost.

+1 on improving the output overall at some point 🙂